packages/ui-common/src/components/button/button.module.scss

```diff
@@ -49,6 +63,9 @@
     background-color: #17a2b8;
     color: white;
   }
+  &-info:active {
+    background-color: #117a8b; /* Darker shade of info color */
```

Comment:
use the darken function. And using scss variables for things like colors will make style tweaking easier later on.
https://falkus.co/2017/05/using-lighten-and-darken-in-sass/